src/c++/perf_analyzer/docs/llm.md

```diff
@@ -53,144 +44,85 @@ docker build -t tritonserver_vllm .
-
-#### 1. Run the following commands to set `max_tokens` to `1`
-
-```bash
-# in the `tutorials/Quick_Deploy/vLLM` directory from above
-PATH_TO_MODEL_PY="model_repository/vllm/1/model.py"
-MAX_TOKENS=1
-sed -i "128s/.*/\ \ \ \ \ \ \ \ params_dict[\"max_tokens\"] = ${MAX_TOKENS}/" ${PATH_TO_MODEL_PY}
-```
-
-#### 2. Start Triton Server
-
+Upon successful build, run the following command to start the Triton Server container:
 ```bash
 docker run --gpus all -it --rm -p 8001:8001 --shm-size=1G --ulimit memlock=-1 --ulimit stack=67108864 -v ${PWD}:/work -w /work tritonserver_vllm tritonserver --model-store ./model_repository
-# this will run continuously in the current shell
 ```
 
-#### 3. Generate prompts input data JSON
-
+Next run the following command to start the Triton SDK container:
 ```bash
-# open a new shell in the same directory you were in when running the above command
-echo '
-{
-    "data": [
-        {
-            "PROMPT": [
-                "Hello, my name is"
-            ],
-            "STREAM": [
-                true
-            ]
-        }
-    ]
-}
-' > prompts.json
+git clone https://github.com/triton-inference-server/client.git
+cd client/src/c++/perf_analyzer/docs/examples
+docker run --gpus all -it --rm --net host -v ${PWD}:/work -w /work nvcr.io/nvidia/tritonserver:23.09-py3-sdk
 ```
 
-#### 4. Run Perf Analyzer
+## Benchmark 1: Profiling the Prefill Phase
 
-```bash
-perf_analyzer \
-    -m vllm \
-    -i grpc \
-    --async \
-    --streaming \
-    --input-data=prompts.json \
-    --profile-export-file=profile_export.json \
-    --measurement-mode=count_windows \
-    --measurement-request-count=10 \
-    --stability-percentage=999
-```
+In this benchmarking scenario, we want to measure the effect of input prompt
+size on first-token latency. We issue single request to the server of fixed
+input sizes and request the model to compute at most one new token. This
+essentially means one pass through the model.
 
-#### 5. Calculate average first-token latency
+#### Example
+
+Inside the client container, run the following command to generate dummy prompts
+of size 100, 300, and 500 and receive single token from the model for each prompt.
 
 ```bash
-python3 examples/calculate_avg_first_token_latency.py
-# Average first-token latency: 0.3065654714375 s
+python profile.py -m vllm --prompt-size-range 100 500 200 --max-tokens 1
+
+# Sample output
+# [ Benchmark Summary ]
+#   Prompt size: 100, Average first-token latency: 0.0459 sec, Average token-token latency: 0.0007 sec
+#   Prompt size: 300, Average first-token latency: 0.0415 sec, Average token-token latency: 0.0007 sec
+#   Prompt size: 500, Average first-token latency: 0.0451 sec, Average token-token latency: 0.0006 sec
 ```
 
-#### 6. Repeat steps 3-5 with different prompt lengths to measure effects of initial prompt size (prefill) on first-token latency.
+> **Note**
+>
+> In order to provide a specific prompt (instead of the dummy prompt generated by default),
```

Comment:
What does the script do in this case? Does it say that it's trying different prompt lengths (when it's really not)? Does it print an error/warning?